config/default/common/config/metadata/stories/geostationary/step002.md

```diff
@@ -0,0 +1,3 @@
+Worldview now has imagery from several geostationary satellites. These satellites follow the Earth's rotation and travel at the same rate of the rotation, so from Earth, it appears the satellite is fixed in one location. This means the satellite captures the same view of Earth and provides almost continuous coverage of one area. Worldview has imagery from the Geostationary Operational Environmental Satellites (GOES)-East, GOES-West, and Himawari-8. The geostationary imagery is available every 10 minutes and on a rolling 90 day basis.
```

Comment:
"(GOES)-East, GOES-West, and Himawari-8"

Either parentheses for both East/West or neither